### PR TITLE
Add feature/more_fs_types and feature/better_alignment

### PR DESCRIPTION
**Pull request by bionix**
- To support more filesystem types like xfs, btrfs, etc.
- To use a configurable and smarter alignment method
- Extend the documentation to support the new features

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -28,6 +52,12 @@
   with_items: '{{ disk_additional_disks }}'
   tags: ['disk']
 
+- name: "Disable periodic fsck on ext3 or ext4 formatted disks"
+  shell: tune2fs -c0 -i0 {{ item.disk }}1
+  with_items: '{{ disk_additional_disks }}'
+  when: "disk_additional_disks and ( item.fstype == 'ext4' or item.fstype == 'ext3' )"
```
Comment (MiLk) [inline tasks/main.yml]: Shouldn't it be configurable by a new option? (which can disable the periodic fsck by default)

File: tasks/main.yml [hunk 2]
```diff
@@ -6,12 +6,36 @@
...
+- name: "Install additional fs progs"
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items: "{{ additional_fs_utils|default([]) }}"
+  when: additional_fs_utils is defined
+  tags: ['disk', 'pkgs']
+
+- name: "Get disk alignment for disks"
+  shell: |
```
Comment (MiLk) [inline tasks/main.yml:18]: Maybe writing a small module instead of using `shell`  would make it easier to maintain in the future.
what do you think?

**Review by MiLk — COMMENTED** [review]
Thank you for your contribution.

As I lack test infrastructure and time to test this role, it might take me some time to get it merged, but I will do my best.

File: tasks/main.yml (hunk 1 above)
Comment (bionix) [reply tasks/main.yml]: I applied an option to control the task `disable_periodic_fsck` (boolean) within the array `disk_additional_disks` .
@MiLk It is good enough for you?

File: tasks/main.yml (hunk 2 above)
Comment (bionix) [reply tasks/main.yml:18]: @MiLk Yes, it makes sense, but my python skills are too low to develop an ansible module. :-/

File: tasks/main.yml (hunk 1 above)
Comment (MiLk) [reply tasks/main.yml]: Perfect!

File: tasks/main.yml (hunk 2 above)
Comment (MiLk) [reply tasks/main.yml:18]: Ok, it will be for another PR then.

**Pull request merged**